packages/common/src/HardhatConfig.js

```diff
@@ -50,6 +52,19 @@ function getHardhatConfig(configOverrides, workingDir = "./", includeTruffle = t
       mumbai: { chainId: 80001, url: getNodeUrl("polygon-mumbai", true), accounts: { mnemonic } },
       matic: { chainId: 137, url: getNodeUrl("polygon-matic", true), accounts: { mnemonic } },
       mainnet: { chainId: 1, url: getNodeUrl("mainnet", true), accounts: { mnemonic } },
+      optimism: {
+        url: "http://127.0.0.1:8545",
+        accounts: { mnemonic: "test test test test test test test test test test test junk" },
+        // This sets the gas price to 0 for all transactions on L2. We do this because account balances are not yet
+        // automatically initiated with an ETH balance.
+        gasPrice: 0,
+        // This sets the network as using the ovm and ensure contract will be compiled against that.
+        ovm: true,
+        // We use custom logic to only compile contracts within the listed directories, as opposed to choosing which
+        // ones to ignore, because there are more contracts to ignore than to include.
+        compileWhitelist: ["oracle/implementation/Finder.sol"],
```

Comment:
looks good. for now we just make _one_ working compilation and can expand later.